com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -78,6 +78,85 @@ public function upgrade_1006() {
     return TRUE;
   }
 
+  /**
+   * Updates job role having wrong funder id with correct contact id
```

Comment:
Please explain here why some job roles have a wrong funder and which rules are used to decide what is the correct contact ID